hadoop-tools/hadoop-aws/pom.xml

```diff
@@ -331,6 +336,44 @@
       </properties>
     </profile>
 
+    <!-- Directory marker retention options, all from the -Dmarkers value-->
+    <profile>
+      <id>keep-markers</id>
+      <activation>
+        <property>
+          <name>markers</name>
+          <value>keep</value>
+        </property>
+      </activation>
+      <properties >
+        <fs.s3a.directory.marker>keep</fs.s3a.directory.marker>
```

Comment:
The property name must be `fs.s3a.directory.marker.retention` to make it consistent with Constants.java.

Reply:
good catch. I'd just renamed the constants and forgot about that